sdk/va-report-components/documentation/docs/api/setLocale.md

```diff
@@ -0,0 +1,30 @@
+---
+id: setLocale
+title: setLocale
+---
+
+```
+setLocale(locale): void
+```
+
+`setLocale` allows the user to set an override to the language specified in the browser settings.
```

Comment:
Small tweaks to help with L10N: ...enables the user to set an override for the language that is specified in the browser settings.